content/en/docs/setup/platform-setup/openshift/index.md

```diff
@@ -87,7 +87,7 @@ $ master-restart api
 $ master-restart controllers
 {{< /text >}}
 
-## Bookinfo
+## Privileged security context constraints for sidecars
 
 The Istio sidecar injected into each pod runs with user ID 1337, which is not allowed by default in OpenShift. To allow this user ID to be used, execute the following commands. Replace `-n bookinfo` with the appropriate namespace.
```

Comment:
Maybe it would be better to replace `-n bookinfo` with `-n app-namespace` here and in the commands?

Reply:
I changed it to `<target-namespace>` to be consistent with the previous section. There is another scc grant command in the previous section that should likely be put together with these ones. Maybe someone with OpenShift knowledge should weigh in.